src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -33,7 +33,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DeployExtension>True</DeployExtension>
-    <CodeAnalysisRuleSet>Integration.Vsix.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>..\Integration.Vsix.UnitTests\Integration.Vsix.UnitTests.ruleset</CodeAnalysisRuleSet>
```

Comment:
This change is not relevant for this PR. Also, it's strange to reference a unittest ruleset in a non-unit test project.